### DISCUSSION TITLE
Sorting function

### DISCUSSION THREAD

**Discussion by luizvnegrini** [wwwdata/implicitly_animated_reorderable_list#17] [Ideas]
Today I need a reorderable list and the only package I found support for something similar was this one, congratulations for the job!

My suggestion is so that we can pass a function for ordering after activating the handle that already exists, my idea is to click and hold the item and when it is released it is ordered according to a specific function and using the animations.

**Comment by wwwdata** [comment]
Can you describe again what you want to do? So the user is somehow long pressing and holding an item, and then releases it without moving? Or is the user moving it to a new position but it then is not at the new position and moves somewhere else? Why would you want to do that?

**Comment by luizvnegrini** [comment]
Sure, I'll try to be clearer.

It's actually not a problem with the current version, it's a suggestion. I need to make a function to pin some items at the beginning of the list, and this pinning would be done by clicking on the item and holding it (it couldn't even let the user drag it to the side), we could have a sort function for that and a flag or aldo of type to lock the slide of the item. The animation would have to zoom in front of the other items and stop in the exact spot after sorting.
